Source/Components/ImageGlass.Heart/Photo.cs

```diff
@@ -707,6 +707,9 @@ public static class Photo {
                 case ".JP2":
                     mimeType = "image/jpeg";
                     break;
+                case ".JXL":
+                    mimeType = "image/jxl";
+                    break;
```

Comment:
I think there is no MIME type for `image/jxl`, maybe just use `image/jpeg`

Reply:
Can be changed sure.

But at least it is used here in official repository:

https://gitlab.com/wg1/jpeg-xl/-/blob/master/plugins/gdk-pixbuf/jxl.thumbnailer

and here:
https://github.com/h5bp/server-configs-apache/pull/256/files
https://github.com/novomesk/qt-jpegxl-image-plugin/blob/main/src/jpegxl.json

and here:
https://lists.w3.org/Archives/Public/ietf-http-wg/2020JulSep/0108.html

`JXL has many more features as a codec than what the lossless JPEG
recompression can offer, you would get a lot better compression density for
similar visual quality if you don't need to be able to reconstruct the
exact same original JPEG file (or if your original was not a JPEG to start
with but RAW data from a camera), so it makes a lot of sense to add jxl as
an image codec on its own.`